nibabel/nicom/dicomwrappers.py

```diff
@@ -21,6 +21,14 @@
 from ..openers import ImageOpener
 from ..onetime import setattr_on_read as one_time
 
+try:
```

Comment:
I ended up writing something of the sort for pr/296 - see : https://github.com/kastman/nibabel/pull/1/commits/2065503e9566855a156c34a97d131db73975726c